OpenBots.Studio/UI/Forms/Supplement Forms/frmAbout.cs

```diff
@@ -40,13 +40,15 @@ private void frmAbout_Load(object sender, EventArgs e)
             lblMacAddress.Text = $"Mac Address: {SystemInfo.GetMacAddress()}";
 
             if (EnvironmentSettings.GetEnvironmentVariable() == null)
+            {
                 lblServer.Text = "Server: Agent environment variable not found";
+                return;
+            }
 
             string agentSettingsPath = Path.Combine(EnvironmentSettings.GetEnvironmentVariable(), EnvironmentSettings.SettingsFileName);
 
             if (agentSettingsPath == null || !File.Exists(agentSettingsPath))
```

Comment:
`agentSettingsPath` cannot be null in this case. It can be removed.
```suggestion
            if (!File.Exists(agentSettingsPath))
```